src/en/news/blog/2023/v17-2-6-quincy-released/index.md

```diff
@@ -0,0 +1,1038 @@
+---
+title: "v17.2.6 Quincy released"
+date: "2023-04-07"
+author: "yuriw"
+tags:
+  - "release"
+  - "quincy"
+---
+
+This is the sixth backport release in the Quincy series. We recommend all users update to this release.
+
+## Notable Changes
+
+- `ceph mgr dump` command now outputs `last_failure_osd_epoch` and
+  `active_clients` fields at the top level.  Previously, these fields were
+  output under `always_on_modules` field.
+
+## Changelog
+
+- msg/async: don't abort when public addrs mismatch bind addrs ([pr#50575](https://github.com/ceph/ceph/pull/50575),         Radoslaw Zarzynski)
+
+- rgw: rgw_parse_url_bucket() rejects empty bucket names after 'tenant:' ([pr#50625](https://github.com/ceph/ceph/pull/        50625), Casey Bodley)
+
+- os/bluestore: Improve deferred write decision ([pr#49333](https://github.com/ceph/ceph/pull/49333), Adam Kupczyk, Igor Fedotov)
+
+- rgw/cloud-transition: Fix issues with MCG endpoint ([pr#49061](https://github.com/ceph/ceph/pull/49061), Soumya Koduri)
+
+- Add per OSD crush\_device\_class definition ([pr#50444](https://github.com/ceph/ceph/pull/50444), Francesco Pantano)
+
+- ceph-crash: drop privleges to run as "ceph" user, rather than root (CVE-2022-3650) ([pr#48805](https://github.com/ceph/ceph/pull/48805), Tim Serong, Guillaume Abrioux)
+
+- ceph-dencoder: Add erasure\_code to denc-mod-osd's target\_link\_libraries ([pr#48028](https://github.com/ceph/ceph/pull/48028), Tim Serong)
+
+- ceph-exporter: cephadm changes ([pr#49771](https://github.com/ceph/ceph/pull/49771), Avan Thakkar)
+
+- ceph-mixing: fix ceph\_hosts variable ([pr#48934](https://github.com/ceph/ceph/pull/48934), Tatjana Dehler)
+
+- ceph-volume/tests: add allowlist\_externals to tox<span></span>.ini ([pr#49788](https://github.com/ceph/ceph/pull/49788), Guillaume Abrioux)
+
+- ceph-volume/tests: fix lvm centos8-filestore-create job ([pr#48122](https://github.com/ceph/ceph/pull/48122), Guillaume Abrioux)
+
+- ceph-volume: add a retry in util<span></span>.disk<span></span>.remove\_partition ([pr#47989](https://github.com/ceph/ceph/pull/47989), Guillaume Abrioux)
+
+- ceph-volume: do not raise RuntimeError in util<span></span>.lsblk ([pr#50144](https://github.com/ceph/ceph/pull/50144), Guillaume Abrioux)
+
+- ceph-volume: fix a bug in get\_all\_devices\_vgs() ([pr#49453](https://github.com/ceph/ceph/pull/49453), Guillaume Abrioux)
+
+- ceph-volume: fix a bug in lsblk\_all() ([pr#49868](https://github.com/ceph/ceph/pull/49868), Guillaume Abrioux)
+
+- ceph-volume: legacy\_encrypted() shouldn't call lsblk() when device is 'tmpfs' ([pr#50161](https://github.com/ceph/ceph/pull/50161), Guillaume Abrioux)
+
+- ceph<span></span>.spec<span></span>.in: disable system\_pmdk on s390x for SUSE distros ([pr#48522](https://github.com/ceph/ceph/pull/48522), Tim Serong)
```

Comment:
Why do we have the `<span></span>` tags here? Looks like HTML's junk. Are we trying to escape the dots?

Reply:
this is done via `ceph-release-notes` with `--markdown` option